lib/raml/parser/response.rb

```diff
@@ -23,6 +23,8 @@ def parse_attributes
           attributes.each do |key, value|
             key = underscore(key)
             case key
+            when 'description'
+              response.send("#{key}=".to_sym, value)
```

Comment:
I think that `response.send` makes sense when matching on the multiple `*BASIC_ATTRIBUTES` - but given that you're just dealing with `description` here I think you should just use `response.description = value`.